src/crawlers/cheerio_crawler.js

```diff
@@ -415,6 +457,14 @@ class CheerioCrawler extends BasicCrawler {
         this.prepareRequestFunction = prepareRequestFunction;
         this.postResponseFunction = postResponseFunction;
         this.proxyConfiguration = proxyConfiguration;
+        /** @type {Array<Hook>} */
+        this.preNavigationHooks = preNavigationHooks;
+        /** @type {Array<Hook>} */
+        this.postNavigationHooks = postNavigationHooks;
+        /** @type {RequestAsBrowserOptions} */
+        this.defaultGotoOptions = {
+            forceUrlEncoding: false,
```

Comment:
This is interesting. Should we set the `false` here? This means that if we decide to flip the default in `requestAsBrowser`, `CheerioCrawler` would still not encode. Is this desired? I think the expected behavior would be that they behave the same.